model/schema/variation.yaml

```diff
@@ -86,22 +105,20 @@ classes:
       the definition of the transcript or protein isoform used by the source has changed, or if there was an error
       in the source data.
     slots:
+      - hgvs
```

Comment:
@markquintontulloch 
Is there a DTO equivalent of the 'hgvs' slot in the VariantLocationAssociationDTO class?

Reply:
@chris-grove HGVS IDs have historically been generated rather than submitted

Reply:
@markquintontulloch Ah, OK, thanks